pkgs/applications/networking/instant-messengers/mikutter/default.nix

```
@@ -1,82 +1,109 @@
-{ lib, stdenv, fetchurl
-, bundlerEnv, ruby
-, alsaUtils, libnotify, which, wrapGAppsHook, gtk2, atk, gobject-introspection
+{ lib
+, stdenv
+, fetchurl
+, bundlerEnv
+, alsaUtils
+, atk
+, bundix
+, bundler
+, common-updater-scripts
+, curl
+, desktop-file-utils
+, gobject-introspection
+, gtk2
+, jq
+, ruby
+, libnotify
+, which
+, wrapGAppsHook
+, writers
 }:
 
-# how to update:
-# find latest version at: http://mikutter.hachune.net/download#download
-# run these commands:
-#
-# wget http://mikutter.hachune.net/bin/mikutter.4.0.0.tar.gz
-# mkdir mikutter
-# cd mikutter
-# tar xvf ../mikutter.4.0.0.tar.gz
-# find . -not -name Gemfile -exec rm {} \;
-# find . -type d -exec rmdir -p --ignore-fail-on-non-empty {} \;
-# cd ..
-# mv mikutter/* .
-# rm mikutter.4.0.0.tar.gz
-# rm gemset.nix Gemfile.lock; nix-shell -p bundler bundix --run 'bundle lock && bundix'
+let
+  version = "4.1.4";
 
+  gems = bundlerEnv {
+    name = "mikutter-${version}-gems";
```

```suggestion
    name = "mikutter-gems";
```

Otherwise we cannot reuse the gems if the version changes.

---

Fixed.